src/pages/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -31,8 +31,8 @@
 							v-if="selectedGroupDescriptor"
 						>
 							to support
-							{{ selectedGroupDescriptor }}
-						</kv-button>.
+							{{ selectedGroupDescriptor }}.
```

Comment:
Unrelated to this feature but this `.` was wrapping by itself to the next line on certain days and certain amounts in the MG summary text